neurons/miners/bitcoin/funds_flow/graph_search.py

```diff
@@ -32,9 +34,14 @@ def __init__(
             auth=(self.graph_db_user, self.graph_db_password),
         )
 
-    def execute_query(self, network, query):
-        # TODO: Implement this
-        return []
+    def execute_cypher_query(self, query):
```

Comment:
does this work? it should be call with mandatory `terms` parameter no?

Reply:
also `execute_query` should be the name as we won't execute any other type of query.